tests/unittest/llmapi/apps/_test_openai_kv_cache_contamination.py

```
@@ -0,0 +1,513 @@
+                max_tokens=max_tokens,
+                temperature=0.0,
+            )
+        )
+        await asyncio.sleep(cancel_after_s)
+        task.cancel()
+        try:
+            await task
+        except asyncio.CancelledError:
+            pass
+        return None
+    except (asyncio.CancelledError, Exception):
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)